Visual Studio 2017 won't compile project

Our documentation states that VS 2017 can build this project, but it has been my experience that it cannot. I think we should either change the documentation to say that VS 2019 is required, or document some workaround for the problem.

I made an [AppVeyor build](https://ci.appveyor.com/project/ddalcino/cpp-starter-project-nx51h/builds/33038649/job/n20yr3wxkj3s7u9k) using Visual Studio 2017 for this project, and it consistently produces this warning, which causes the build to fail:

    catch.hpp(2167): warning C4619: #pragma warning: there is no warning number '4800'

There's an issue for this filed in Catch2's [github](https://github.com/catchorg/Catch2/issues/1787), but it's closed as a won't-fix. There's an issue filed at [MSDN](https://developercommunity.visualstudio.com/content/problem/78259/visual-studio-2017-no-longer-generates-a-c4800-war.html), but they closed it by saying "We fixed it in VS 2019".

--- a/README.md
+++ b/README.md
@@ -82,12 +82,11 @@ The following compilers should work:
 			brew install llvm
 	</details>
 
-  * [Visual Studio 2017 or higher](https://visualstudio.microsoft.com/)
+  * [Visual Studio 2019 or higher](https://visualstudio.microsoft.com/)
 	<details>
 	<summary>Install command + Environment setup</summary>
 
-	On Windows, you need to install Visual Studio because of the SDK and libraries that ship with it 
-	(the minimum compiler version is 19.15, which ships with version 15.8 of the IDE).
+	On Windows, you need to install Visual Studio 2019 because of the SDK and libraries that ship with it.
 
   	Visual Studio IDE - 2019 Community (installs Clang too):
 		

